Lib/test/test_py3kwarn.py

```diff
@@ -247,19 +247,47 @@ def test_file_xreadlines(self):
             with check_py3k_warnings() as w:
                 self.assertWarning(f.xreadlines(), w, expected)
 
-    def test_file_open(self):
```

Comment:
These tests were deleted in eror I want to treat this as a minor change, where I return them and squash at once. Is this OK @ltratt or I push the tests then you review for a fresh squash request?

Reply:
Push the tests for a fresh review please -- thanks :)